src/pages/Editor.vue

```diff
@@ -1386,6 +1388,9 @@ export default {
     },
   },
   methods: {
+    forceUpdate() {
+      this.componentKey++;
```

Comment:
Not really needed to keep increasing the key. Just changing it from 0 to 1 or 1 to 0 is fine.

Reply:
okayy,will do that.